engine-rest/pom.xml

```diff
@@ -15,7 +15,7 @@
   <name>camunda BPM - engine - REST - Root</name>
 
   <properties>
-    <jackson.version>2.9.8</jackson.version>
+    <jackson.version>2.9.9</jackson.version>
```

Comment:
The REST API should explicitly declare a `jackson-databind` dependency in version 2.9.9.1, or else, all REST artifacts will only use 2.9.9. In addition, it looks like this also results in duplicate `jackson-databind` dependencies in downstream artifacts, such as the standalone webapps.

Reply:
The only place where `jackson.version` is used in the REST API is [here](https://github.com/camunda/camunda-bpm-platform/blob/4361cdbbae0d0cb320e46e3bdabde8d145e3e555/engine-rest/engine-rest/pom.xml#L36-L40), which is for `jackson-jaxrs-json-provider` (no reported vulnerability) with latest version `2.9.9`